src/app/public/modules/definition-list/fixtures/definition-list-fixtures.module.ts

```diff
@@ -14,6 +28,9 @@ import { SkyDefinitionListTestComponent } from './definition-list.component.fixt
   ],
   exports: [
     SkyDefinitionListTestComponent
+  ],
+  providers: [
+    SkyDefinitionListAdapterService
```

Comment:
Is this provider needed since it's included in the `SkyDefinitionListModule`?

Reply:
Nope. Removed.